File: lb_4/src/NPC.java

public class NPC extends Character
{
	protected int lvlFriendly;
	protected String activity;
	
	public NPC(String name, int health){
		super (name, health);
	}
	
	protected String Speak(String activityNPC)
	{
		return activityNPC=="Inhabitant" ? "Hi. Do you want to speak?" : "Hi. Do you want to trade?";
	}

}

File: lb9_A/lb8_A/NPC.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace lb9_A
{
    class NPC: Character
    {
		protected int lvlFriendly;
		protected String activity;

		public NPC (string name, int health)
			: base(name, health)
		{
			
		}
	}
}
